CitationStyleLanguagePlugin.php

```diff
@@ -446,7 +446,7 @@ public function getCitation(PKPRequest $request, Submission $submission, string
             if ($publication->getData('pages')) {
                 $citationData->page = htmlspecialchars($publication->getData('pages'));
             }
-            $citationData->abstract = htmlspecialchars(strip_tags($publication->getLocalizedData('abstract')));
+            $citationData->abstract = htmlspecialchars_decode(strip_tags($publication->getLocalizedData('abstract')));
```

Comment:
The `getLocalizedAbstract()` was removed, but perhaps we should bring it back to life, just to make use of the `$format` and standardize the access to such HTML fields 🤔

Reply:
Just to avoid spreading `htmlspecialchars_decode()` everywhere... Perhaps the usage of this field will also have to be reviewed on the rest of the application o_O

Reply:
I see abstract and other HTML fields/settings (e.g. section policy) are treated with PKPString::html2text. Thus maybe I will not introduce those getLocalized.. functions now, but, as everywhere use PKPString::html2text here. OK?

Reply:
So actually PKPString::html2text and that getLocalizedTitle function are pretty much the same, except that those special elements (e.g. <li>) are not treated in getLocalizedTitle, which is I suppose OK, because the title cannot contain them...